### ISSUE TITLE
Not working with pug?

### ISSUE THREAD

**Issue by pwnjack** [taptapship/wiredep#243]
Wiredep seems to not work with .pug (formerly .jade). Am I doing something wrong?

This is my gulp task:

    gulp.task('wiredep', function () {
      gulp.src('app/views/partials/master.pug')
        .pipe(wiredep({ignorePath: '../../'}))
        .pipe(gulp.dest('app/views/partials/'));
    });

And these are the tags in my .pug file:

    //- bower:css
    //- endbower

    //- bower:js
    //- endbower

These are the dependencies that should be injected, from my bower.json file.

    "dependencies": {
        "bootstrap": "^3.3.6",
        "font-awesome": "^4.6.3",
        "holderjs": "^2.9.3"
      },

**Comment by eddiemonge** [comment]
thats because pug support hasnt been added in yet https://github.com/taptapship/wiredep/pull/239

**Comment by pwnjack** [comment]
I noticed the merge and I downloaded wiredep again via npm, the injection still not working with pug files, tho. Any clue?

Edit: the package on npm registry seems not updated yet, that's why.

**Comment by eddiemonge** [comment]
Yeah I didn't do a release yet since its needs a bit of testing. If you want to npm install the git version and try it out that would be helpful.

**Comment by daviddiasfront** [comment]
Hi, I tried wiredep (gulp-wiredep) with PUG files, but it's still not working... With Jade is fine, but if I change .jade to .pug with the same code, my generated file don't include bower plugin's paths.

**Comment by carusog** [comment]
@pwnjack
You can use HTML comments as usual waiting for a full integration since all lines beginning with `<` are treated as plain text by Pug, so

``` html
<!--bower:css-->
<!--endbower-->
```

will work just fine.

**Comment by pwnjack** [comment]
@carusog 
Nice workaround.
For the records, i tried out the new version via git install and after a bit of testing it seems to work fine! ;)

**Comment by zhekaus** [comment]
I've spend two hours to figure out what is wrong with wiredep (((

**Comment by Tenemo** [comment]
@carusog
> You can use HTML comments as usual waiting for a full integration since all lines beginning with < are treated as plain text by Pug, so
```
<!--bower:css-->
<!--endbower-->
```
> will work just fine.

Thanks, that works perfectly! To anyone who's stuck at this as long as I was - carusog's fix still works and is required, because wiredep, as of 4.0.0, doesn't support .pug. The only downside is that now the html comment will actually appear in the output, unlike pug comment lines, but I can live with that.

**Comment by pwnjack** [comment]
@Tenemo 

You can install the latest wiredep version (working with pug) via npm with this command:

    npm install git+https://github.com/taptapship/wiredep.git

This is the git repo version and will work just fine.
